server/config/passport.js

```diff
@@ -38,8 +38,7 @@ module.exports = passport.use(
     },
     async (payload, done) => {
       try {
-        const user = await User.findById(payload.user._id);
-        user ? done(null, user) : done(null, false);
+        payload.user ? done(null, payload.user) : done(null, false);
```

Comment:
The entire user is currently stored in the JWT so it felt like a redundant step to hit the database on every request if its not needed. Unsure if this is bad practice but seems like a more lightweight approach.

Reply:
Ah ok. If the entire user object is being stored it'd not necessary. Typically you'd only want a single thing like the id in the token. If you store the whole object, then the user updates something that's stored on the object but the token object is what's referenced, that could cause issues. But for our use case this will ok